hadoop-ozone/dist/src/main/license/jar-report.txt

```diff
@@ -25,33 +29,43 @@ share/ozone/lib/commons-configuration2.jar
 share/ozone/lib/commons-daemon.jar
 share/ozone/lib/commons-digester.jar
 share/ozone/lib/commons-io.jar
+share/ozone/lib/commons-lang.jar
 share/ozone/lib/commons-lang3.jar
 share/ozone/lib/commons-logging.jar
 share/ozone/lib/commons-math3.jar
 share/ozone/lib/commons-net.jar
 share/ozone/lib/commons-pool2.jar
 share/ozone/lib/commons-text.jar
 share/ozone/lib/commons-validator.jar
+share/ozone/lib/compiler.jar
 share/ozone/lib/derby.jar
 share/ozone/lib/disruptor.jar
 share/ozone/lib/dnsjava.jar
+share/ozone/lib/elasticsearch-cli.jar
+share/ozone/lib/elasticsearch-core.jar
+share/ozone/lib/elasticsearch-geo.jar
+share/ozone/lib/elasticsearch-rest-client.jar
+share/ozone/lib/elasticsearch-rest-high-level-client.jar
+share/ozone/lib/elasticsearch-secure-sm.jar
+share/ozone/lib/elasticsearch-x-content.jar
+share/ozone/lib/elasticsearch.jar
 share/ozone/lib/error_prone_annotations.jar
 share/ozone/lib/failureaccess.jar
-share/ozone/lib/FastInfoset.jar
+share/ozone/lib/gethostname4j.jar
 share/ozone/lib/grpc-api.jar
 share/ozone/lib/grpc-context.jar
 share/ozone/lib/grpc-core.jar
 share/ozone/lib/grpc-netty.jar
-share/ozone/lib/grpc-protobuf.jar
 share/ozone/lib/grpc-protobuf-lite.jar
+share/ozone/lib/grpc-protobuf.jar
 share/ozone/lib/grpc-stub.jar
 share/ozone/lib/gson.jar
 share/ozone/lib/guava-jre.jar
 share/ozone/lib/guice-assistedinject.jar
 share/ozone/lib/guice-bridge.jar
-share/ozone/lib/guice.jar
 share/ozone/lib/guice-multibindings.jar
 share/ozone/lib/guice-servlet.jar
+share/ozone/lib/guice.jar
```

Comment:
nit: can we avoid re-ordering the lines that indicates the same jar and keep only the lines added (or removed if any)

Reply:
This file is updated automatically by running `update-jar-report.sh`. I believe the order changes due to system specific implementations of `sort`, which determines things like whether of not `share/ozone/lib/guice.jar` comes before or after `share/ozone/lib/guice-multibindings.jar`.